SwappinessCheck.pm

```diff
@@ -58,11 +58,11 @@ sub execute
 			$self->{RESULTKIND} = "good";
 		}
 	}
-	else
-	{
-		$self->{RESULTKIND} = "warning";
-		print "warning: '/sbin/sysctl vm.swappiness' did not produce a parsable result\n";
-	}
+#	else
+#	{
+#		$self->{RESULTKIND} = "warning";
+#		print "warning: '/sbin/sysctl vm.swappiness' did not produce a parsable result\n";
+#	}
```

Comment:
I think we should keep this warning, and it can remain as it was: we will still be using `sysctl` to inspect the current value of this setting, even though we won't suggest using it to modify the value anymore.